### PR TITLE
Refactor to NuxtJS

### PR DESCRIPTION
**Pull request by marc-chouinard**
Refactor/redesign of the entire site to NuxtJS. Should make for easier updating in the future.

### CODE REVIEW

File: assets/css/mobile-top-nav.scss [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+		height: 4px;
+		margin-bottom: 5px;
+		position: relative;
+		background: #ffffff;
+		border-radius: 3px;
+		z-index: 1;
+		transform-origin: 4px 0px;
+		transition: transform 0.6s cubic-bezier(0.77, 0.2, 0.05, 1.0),
+		background 0.6s cubic-bezier(0.77, 0.2, 0.05, 1.0),
+		opacity 0.6s ease;
```
Comment (6102bitcoin) [inline assets/css/mobile-top-nav.scss:77]: After some tweaking I think this looks smoother. 
```
		transition: transform 0.6s cubic-bezier(0.77, 0.2, 0.05, 1.0),
		background 0.6s cubic-bezier(0.77, 0.2, 0.05, 1.0),
		opacity 0.6s ease;
```

File: assets/css/mobile-top-nav.scss [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+	}
+
+	.mobile-menu-toggle input:checked ~ span {
+		opacity: 1;
+		transform: rotate(45deg) translate(4px, -2px);
+	}
+
+	.mobile-menu-toggle input:checked ~ span:nth-last-child(2) {
+		opacity: 0;
+		transform: rotate(0deg) scale(0.2, 0.2);
```
Comment (6102bitcoin) [inline assets/css/mobile-top-nav.scss:99]: Recommend changing to below 
```
transform: rotate(0deg) scale(0.0, 0.0);
transition: transform cubic-bezier(0.16, 1, 0.3, 1)
 ```

File: pages/about.vue [hunk 3]
```diff
@@ -0,0 +1,61 @@
+<template>
+	<div class="about-page">
+
+		<h1 class="page-title">About</h1>
+
+		<p class="about-me">My name is <a href="https://twitter.com/6102bitcoin" target="_blank">@6102bitcoin</a> and I created this website to showcase the highest quality Bitcoin resources. I'm a pseudonymous Bitcoin user and educator doing what I can to accelerate Bitcoin understanding.</p>
```
Comment (6102bitcoin) [inline pages/about.vue:6]:
```
<p class="about-me">Bitcoin only is an open source website created to showcase the highest quality Bitcoin resources. It is built and maintained by volunteers.</p>
```

File: pages/privacy.vue [hunk 4]
```diff
@@ -0,0 +1,267 @@
+<template>
+	<div id="privacy-page">
+
+		<h1 class="page-title">Privacy</h1>
+
+		<div class="getting-started-section">
+
+			<h3>Getting Started</h3>
+
+			<p>Read my FAQ on how to <a href="https://github.com/6102bitcoin/FAQ/blob/master/hodl-privacy.md" target="_blank">hold Bitcoin privately</a>.</p>
```
Comment (6102bitcoin) [inline pages/privacy.vue:10]: Remove broken link
```
			<p>Don't reuse addresses, learn to CoinJoin and use your own node. </p>
			<p>Read how to <a href="https://bitcoinprivacy.guide/" target="_blank">use Bitcoin Privately & <a href="https://en.bitcoin.it/Privacy" target="_blank">the bitcoin wiki page</a></p>
```

File: pages/promote.vue [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+		<h3>In Meatspace</h3>
+
+		<p>Want to help promote bitcoin-only.com? The gorgeous poster below was designed by Phneep.</p>
+
+		<ul class="poster-instructions">
+			<li><span>1)</span>Print it out</li>
+			<li><span>2)</span>Put it up somewhere busy</li>
+			<li><span>3)</span>Reply to this tweet with an in-context photo</li>
+		</ul>
```
Comment (6102bitcoin) [inline pages/promote.vue:19]: Make more clear
```
		<p>Want to help promote bitcoin-only.com? The poster below was designed by Phneep.</p>

		<ul class="poster-instructions">
			<li><span>1)</span>Print it out</li>
			<li><span>2)</span>Put it up somewhere public</li>
			<li><span>3)</span>Take a photo of the poster and tag <a href="https://twitter.com/bitcoinonly_" target="_blank">@bitcoinonly_</a> on twitter</li>
		</ul>
```

File: pages/wallets.vue [hunk 6]
```diff
@@ -0,0 +1,448 @@
...
+		<h1 class="page-title">Wallets</h1>
+
+		<div class="getting-started-section">
+
+			<h3>Getting Started</h3>
+
+			<ul>
+				<li>Desktop: Download <a href="https://bitcoin.org/" target="_blank">Bitcoin Core</a> & <a href="https://github.com/cryptoadvance/specter-desktop" target="_blank">Specter Desktop</a></li>
+				<li>iOS: Download <a href="https://bluewallet.io/" target="_blank">Blue Wallet</a></li>
+				<li>Android: Download <a href="https://samouraiwallet.com/" target="_blank">Samourai Wallet</a></li>
```
Comment (6102bitcoin) [inline pages/wallets.vue:13]: Remove redundant 'download' instruction 
```
				<li>Desktop: <a href="https://bitcoin.org/" target="_blank">Bitcoin Core</a> & <a href="https://github.com/cryptoadvance/specter-desktop" target="_blank">Specter Desktop</a></li>
				<li>iOS: <a href="https://bluewallet.io/" target="_blank">Blue Wallet</a></li>
				<li>Android: <a href="https://samouraiwallet.com/" target="_blank">Samourai Wallet</a></li>
```

File: pages/wallets.vue [hunk 7]
```diff
@@ -0,0 +1,448 @@
...
+			</ul>
+
+		</div>
+
+		<p>The wallets listed on this page have explicitly stated that they support Bitcoin only.</p>
+
+		<p>It is very important that you learn how to backup your mnemonic seed & that if you use a passphrase you back that up too. Read <a href="https://github.com/6102bitcoin/FAQ/blob/master/seed.md" target="_blank">this document</a> to learn more about backing up your wallet and check out the <nuxt-link to="/privacy">privacy page</nuxt-link>. See <a href="https://veriphi.io/en/blog/software-wallet-analysis" target="_blank">this report</a> for a comparison of the features of many of the wallets listed below.</p>
```
Comment (6102bitcoin) [inline pages/wallets.vue:20]: Fix broken link
```
		<p>It is very important that you learn how to backup your mnemonic seed & that if you use a passphrase you back that up too. Read <https://bitcoin-intro.com/en/backup" target="_blank">this document</a> to learn more about backing up your wallet and check out the <nuxt-link to="/privacy">privacy page</nuxt-link>.</p>
		<p> See <a href="https://veriphi.io/en/blog/software-wallet-analysis" target="_blank">this report</a> for a comparison of the features of many of the wallets listed below.</p>
```

**Review by 6102bitcoin — COMMENTED** [review]
A number of suggested changes

**Comment by 6102bitcoin** [comment]
ACK

**Review by wiz — APPROVED** [review]
ACK

**Pull request merged**